crates/biome_js_type_info/snapshots/infer_flattened_type_from_direct_promise_instance.snap

```diff
@@ -0,0 +1,22 @@
+---
+source: crates/biome_js_type_info/tests/utils.rs
+expression: content
+---
+```ts
+new Promise((resolve) => resolve("value"));
+
+```
+
+```
+instanceof Promise
+```
+
+## Registered types:
+
+Global TypeId(6) => sync Function {
```

Comment:
Out of curiosity, what are the other five types?

Reply:
They're predefined types defined in `globals.rs`. They'll always be the same ones, so I omit them from the snapshots.